pkg/kubernetes-utils_test.go

```diff
@@ -0,0 +1,30 @@
+package keptnkubeutils
+
+import (
+	"testing"
+
+	"github.com/magiconair/properties/assert"
+	"helm.sh/helm/v3/pkg/chart/loader"
+)
+
+var recommendedLabelsCharts = []struct {
+	chartPath      string
+	expectedResult bool
+}{
+	{"testdata/all_recommended_labels", true},
+	{"testdata/some_recommended_labels", false},
```

Comment:
I used `helm create` to generate charts. These charts had all but 2 of the recommended labels (`app.kubernetes.io/component`, `app.kubernetes.io/part-of`) which I added by myself.